redaxo/src/core/lib/login/backend_login.php

```diff
@@ -154,9 +156,20 @@ public function checkLogin()
             }
         }
 
+        // check if session was killed only if the user is logged in
+        if ($check) {
+            $sql->setQuery('SELECT 1 FROM '.rex::getTable('user_session').' where session_id = ?', [session_id()]);
```

Comment:
In der Core-update.php am besten die laufende Session des Admins in die neue Tabelle eintragen, damit er nach Core-Update auf 5.15 nicht ausgeloggt wird.

Reply:
fix in https://github.com/redaxo/redaxo/pull/5376